test/pkg.jl

```diff
@@ -100,6 +100,8 @@ temp_pkg_dir() do
     str = chomp(takebuf_string(iob))
     @test endswith(str, string(Pkg.installed("Example")))
     @test isempty(Pkg.dependents("Example"))
+    # Checking to see dependency reading
+    @test Pkg.dependencies("Example") == Dict("julia" => Pkg.Types.VersionSet(v"0.3.0", typemax(VersionNumber)))
```

Comment:
while it's not impossible that some future version of REPL.jl may be resurrected and un-deprecated, I think it's a lot less likely than Example.jl needing to be updating because of some breaking change in how packages are structured or what information they should contain.